lib/vm/src/memory.rs

```diff
@@ -445,6 +493,11 @@ impl LinearMemory for VMMemory {
     unsafe fn initialize_with_data(&self, start: usize, data: &[u8]) -> Result<(), Trap> {
         self.0.initialize_with_data(start, data)
     }
+
+    /// Copies this memory to a new memory
+    fn fork(&mut self) -> Result<Box<dyn LinearMemory + 'static>, MemoryError> {
```

Comment:
Let's use clone. I think is a more accurate naming

Reply:
Renamed to `duplicate()`.